vyxal/elements.py

```diff
@@ -1739,12 +1740,16 @@ def gen_from_fn(lhs, rhs, ctx):
     def gen():
         yield from lhs
 
-        made = lhs
+        made = []
+        made += lhs
```

Comment:
since you imported `copy`, you could just do this
```suggestion
        made = copy.copy(lhs)
```

or

```suggestion
        made = [*lhs]
```

Reply:
*Pointer, is that you?!*